server/storage/mvcc/key_index.go

```diff
@@ -329,6 +329,12 @@ func (ki *keyIndex) equal(b *keyIndex) bool {
 
 func (ki *keyIndex) String() string {
 	var s string
+	if len(ki.key) > 128 {
+		s += string(ki.key[:128]) + "...\n"
+	} else {
+		s += string(ki.key[:]) + "\n"
+	}
+	s += fmt.Sprintf("modified[%d]\n", ki.modified)
```

Comment:
Not sure what's the reason to change `String` method, however the code:
* Looks more and more inefficient (strings a immutable in Go, so each `+=` does a full copy)
* will not work with zap correctly as newline characters will be escaped.